pre_install_report/README.md

```diff
@@ -1,15 +1,60 @@
 # Pre-Installation Check of SAS Viya System Requirements
+## Overview
+This tool compares your Kubernetes environment to the SAS Viya system requirements.  It evaluates a   
+number of items, such as memory, CPU cores, software versions, and permissions. The output is a web-viewable,  
+HTML report with the results. SAS recommends running the tool and resolving any reported issues before   
+beginning a SAS Viya deployment in a Kubernetes cluster.  
+
+### Memory and vCPU Check
+The tool calculates the aggregate Memory and aggregate vCPUs of all the nodes that must be active and running. The aggregate
+Memory is the sum the Memory capacity on all the running nodes. The aggregate CPU is calculated similarly.
+  
+This calculated aggregate Memory must equal or exceed a percentage(85%) of required aggregate Memory.  The calculated   
+aggregate number of vCPUs must equal or exceed the required aggregate number of vCPUs for your deployment offering. The   
+requirements per offering are detailed in the _Hardware and Resource Requirements_ section of the SAS Viya Operations   
+document. The required Memory and vCPUs sizes depend on the instance type used for the node.  
+
+Your required aggregates must be specified in the following file  
+<tool-download-dir>/viya4-ark/pre_install_report/viya_deployment_settings.ini, example:
+```
+# Total Memory of all worker Nodes in Gib. Sum of the Memory on all active node required to deploy a specific offering.
+# Set value for required for offering
+VIYA_MIN_AGGREGATE_WORKER_MEMORY=448Gi
+# Total CPU of all worker Nodes in millicores.  Sum of the vCPUs on all active node required to deploy a specific offering.
+# Minimum allowed value = '.001'. 
+VIYA_MIN_AGGREGATE_WORKER_CPU_CORES=56
+```
+
+If calculated aggregate memory is less than VIYA_MIN_AGGREGATE_WORKER_MEMORY then the tool will flag a memory issue.  
+If calculated aggregate vCPUs is less than VIYA_MIN_AGGREGATE_WORKER_CPU_CORES then it the tool will flag a CPU issue.
+ 
+SAS recommends using the SAS Viya 4 Infrastructure as Code (IaC) tools to create a cluster for Microsoft Azure, AWS,   
```

Comment:
Can you embed the links to the IaC GitHub repos like so... 
"SAS recommends using SAS Viya 4 Infrastructure as Code (IaC) tools to create a cluster for [Microsoft Azure](https://github.com/sassoftware/viya4-iac-azure), [AWS](https://github.com/sassoftware/viya4-iac-aws), or [GCP](https://github.com/sassoftware/viya4-iac-gcp)"

Reply:
If I copy this as is I don't get the links in the README.

Reply:
Making needed changes.  Please review after push